### DISCUSSION TITLE
Converting a Strategy into a live strategy ordering on exchanges

### DISCUSSION THREAD

**Discussion by pazooki** [kernc/backtesting.py#947] [Q&A] [answered]
Hi there, 

I want to move forward with a strategy to set it up to trade live. What's the easiest way to integrate `Backtesting.py` with a restful exchange api to let it trade? and is there a way to keep appending data to `self.data` ? 

if there is a way to add the incoming data from websocket of an exchange to `self.data` then I think most of the problem of integration gets resolved. what are your thoughts on this ?

**Comment by kernc** [comment, marked as answer]
Did you see https://github.com/kernc/backtesting.py/issues/81?
